e2e/nomostest/testpredicates/predicates.go

```diff
@@ -817,16 +819,15 @@ func MissingDeletionTimestamp(o client.Object) error {
 }
 
 // ObjectNotFoundPredicate returns an error unless the object is nil (not found).
-func ObjectNotFoundPredicate(nt *NT) Predicate {
-	ntPtr := nt
+func ObjectNotFoundPredicate(scheme *runtime.Scheme) Predicate {
```

Comment:
These schemes all make me think this should be a PredicateFactory, but that seems potentially tedious...